aries-site/src/pages/learn/how-to-add-additional-controls-to-a-toolbar.mdx

```diff
@@ -0,0 +1,191 @@
+import { Box, Image } from 'grommet';
+
+## Prerequisites
+
+This guide builds off the ["How to add search and filter to DataTable" how-to guide](/learn/how-to-add-search-and-filter-to-datatable). While it's not necessary to complete that guide first, it may be helpful for additional context.
```

Comment:
```suggestion
This guide builds off the ["How to add search and filter to DataTable" how-to guide](/learn/how-to-add-search-and-filter-to-datatable-with-data). While it's not necessary to complete that guide first, it may be helpful for additional context.
```
This link isn't working, I think it needs changed to this